joomla/joomla-cms
# [5.1] language code rich snippet revert

### Summary of Changes
In the large Rich Snippets PR #41151 a line was changed that has resulted in invalid markup and broken functionality of the language code plugin


### Testing Instructions
1. Create an article in and view the article in the front end.
Check the source and you will see the invalid markup
`<meta content="en-GB">`

2. open the  language code plugin, enable and save
go to the new tab in that plugin and change the lanbguage code to en-BT
Refresh the front and refresh the page. View the source and you will see the same invalid markup as in test 1

3. Apply the one line change in this PR
Disable the language code plugin
Refresh the front and refresh the page. View the source and you will see the` <meta itemprop="inLanguage" content="en-GB">`
This is the correct and valid markup
4. Re-enable the language code plugin and check the frone end again and you will see `<meta itemprop="inLanguage" content="en-BT">`
### Link to documentations
Please select:
